frontend/src/components/c-stacked-bar-chart.vue

```diff
@@ -0,0 +1,23 @@
+<template lang="pug">
+#summary
+  .summary-chart__contrib--bar(

```

Comment:
Since this is no longer only linked to `summary-charts`, would it be possible to rename the class here?

You might have to update some test cases since I believe they are using the `.summary-chart__contrib--bar` selector.